src/components/Input/ButtonIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Icon } from '@mdi/react';
+import cx from 'classnames';
+
+import Button from '@/components/Input/Button';
+import { buttonSizeClasses, buttonTypeClasses } from '@/components/Input/Button.utils';
+
+import type { ButtonType, SizeType } from '@/components/Input/Button.utils';
+
+type IconButtonProps = {
+  icon: string;
+
+  className?: string;
+  onClick: React.MouseEventHandler<HTMLDivElement | HTMLButtonElement>;
+  buttonType: ButtonType;
+  buttonSize: SizeType;
+};
+
+const ButtonIcon = ({ buttonSize = 'normal', buttonType = 'secondary', className, icon, onClick }: IconButtonProps) => (
```

Comment:
`IconButton` is probably a better name

Reply:
I agree but I wanted to keep the naming convention the same since we have Buttton and ButtonDropdown.

Reply:
That's because I missed it.. That should be `DropdownButton` too